app/models/video.py

```diff
@@ -1,4 +1,21 @@
 from app import db
```

Comment:
I just want to note that this file is almost 400 lines long.  Maybe consider breaking it up into separate files.